mkdocs.yml

```diff
@@ -21,6 +21,7 @@ nav:
     - Testing: testing-be.md
     - Translation: translation.md
     - Caching: caching.md
+    - Housing counselor data processing: housing-counselor-data.md
```

Comment:
We've discussed this a bit as part of a larger documentation IA refactor, but this might be a good opportunity to move this down to the "API" section and rename it to something like "Site content" or "What's on cf.gov" (there's probably a much better name for this). What do you think @Scotchester?

Reply:
I'm not sure. The content of the existing pages in the API section is significantly different than this page; they are about APIs we _provide_. I think I like it better here in the back-end section, for now.